addons/editor-theme3/theme3.js

```diff
@@ -615,6 +615,35 @@ export default async function ({ addon, console, msg }) {
     Object.assign(originalColors, newColors);
   };
 
+  (async () => {
+    // Custom colors for "Add an input/label" block icons in the "Make a block" popup menu, by pumpkinhasapatch
+    const uriHeader = "data:image/svg+xml;base64,";
+    const myBlocksCategory = categories[9];
```

Comment:
This will break if any categories above My Blocks are added or removed in the future. `myBlocksCategory` should be defined as a constant at the top of the file and then referenced in the `categories` array, like the existing `extensionsCategory` and `saCategory`.